src/components/screens/lockedBalance/lockedBalance/balanceTable.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import moment from 'moment';
+import Icon from '../../../toolbox/icon';
+import { fromRawLsk } from '../../../../utils/lsk';
+import { getDelayedAvailability, isBlockHeightReached } from '../../../../utils/account';
+import styles from './lockedBalance.css';
+
+const getPendingTime = ({ unvoteHeight, delegateAddress }, currentBlockHeight, { address }) => {
+  const isSelfVote = address === delegateAddress;
+  const delayedAvailability = getDelayedAvailability(isSelfVote);
+  const awaitingBlocks = delayedAvailability - (currentBlockHeight - unvoteHeight);
+  const secondsToUnlockAllBalance = awaitingBlocks * 10;
+  const momentSeconds = moment().second(secondsToUnlockAllBalance);
+  return moment().to(momentSeconds, true);
+};
+
+const BalanceTable = ({
+  t,
+  lockedBalance,
+  availableBalance,
+  currentBlock,
+  account,
+}) => (
+  <ul className={`${styles.amountStatusContainer} lock-balance-amount-container`}>
+    <li>
+      <p className={styles.columnTitle}>{t('Amount')}</p>
+      <p className={styles.columnTitle}>{t('Status')}</p>
+    </li>
+    <li>
+      <p className="locked-balance">{`${fromRawLsk(lockedBalance)} LSK`}</p>
+      <p>
+        <Icon name="lock" />
+        {t('locked')}
+      </p>
```

Comment:
Show this section only if there are some tokens locked. e.g.` lockedBalance !== 0`. Same goes for the other two parts: will be unlocked and available for unlock.